examples/esp32spi_tcp_client.py

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2021 Adafruit Industries
+# SPDX-License-Identifier: MIT
+
+import board
+from digitalio import DigitalInOut
+from adafruit_esp32spi import adafruit_esp32spi
+import adafruit_esp32spi.adafruit_esp32spi_socket as socket
+import adafruit_requests as requests
+from secrets import secrets
+
+
+TIMEOUT = 5
+# adjust host and port to match server
+HOST = "192.168.10.179"
```

Comment:
Might want to add a different IP or placeholder text/IP here?

Reply:
different IP? I'm happy to dummy it up somehow, not sure what you're thinking?

Reply:
I could use wifitest.adafruit.com and port 80 ...that would give some 404 rexponse or similar

Reply:
`Receiving
b'HTTP/1.1 400 Bad Request\r\nServer: nginx/1.10.3 (Ubuntu)\r\nDate: Mon, 15 Mar 2021 16:24:42 GMT\r\nContent-Type: text/html\r\nContent-L'`